getting-started/mbed_app.json

```diff
@@ -2,7 +2,7 @@
     "target_overrides": {
         "*": {
              "platform.stdio-convert-newlines": true,
-             "target.features_add" : ["EXPERIMENTAL_API"],
+             "target.features_add" : ["EXPERIMENTAL_API", "PSA"],
```

Comment:
Commit message in "Enable PSA by default" is doesn't seem to match what we are doing in that commit.